roles/openshift_prometheus/tasks/install_prometheus.yaml

```diff
@@ -0,0 +1,223 @@
+---
+
+# namespace
+- name: Add prometheus project
+  oc_project:
+    state: "{{ state }}"
+    name: "{{ openshift_prometheus_namespace }}"
+    node_selector: "{{ openshift_prometheus_node_selector | regex_replace(':','=') | regex_replace('{', '[') | regex_replace('}', ']') }}"
```

Comment:
@zgalor it fails to create the project for me with the recent changes. There is a bug in oc_project that was causing the module to run without erroring out (fixed with #5058 ), but even without this fix the project isn't being created. It just makes you believe the project was created.

The next task would fail immediately because the project wasn't created.

With the oc_project fix applied, it gives more useful output:

```
TASK [openshift_prometheus : Add prometheus project] ***************************
Thursday 10 August 2017  20:25:18 +0000 (0:00:00.050)       0:01:42.673 ******* 
fatal: [54.191.165.68]: FAILED! => {
    "changed": false, 
    "failed": true
}

MSG:

{u'returncode': 1, u'cmd': u"/usr/bin/oc adm new-project prometheus --admin-role=admin --description=Prometheus --node-selector=['type'= 'infra']", u'results': {}, u'stderr': u'The Project "prometheus" is invalid: nodeSelector: Invalid value: "[\'type\'= \'infra\']": must be a valid label selector\n', u'stdout': u''}
```